host/xtest/regression_4000.c

```diff
@@ -4860,15 +4860,16 @@ static void xtest_tee_test_4009(ADBG_Case_t *c)
 			       TEE_ATTR_ECC_PRIVATE_VALUE,
 			       pt->private, size_bytes);
 		/*
-		 * The public value is not used. This is why we provide
-		 * another buffer
+		 * The public value is not used, but we should provide a valid
+		 * one to avoid rejection in case TEE_PopulateTransientObject
```

Comment:
suggest to add parentheses to the function label in  the message: `TEE_PopulateTransientObject()`.